packages/create-plugin/fixtures/test-template/src/plugin.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/grafana/grafana/main/docs/sources/developers/plugins/plugin.schema.json",
   "type": "app",
   "name": "{{ titleCase pluginName }}",
-  "id": "{{ normalize_id pluginName orgName 'app' }}",
+  "id": "{{ pluginId }}",
```

Comment:
If `pluginId` is now the source of truth for all templates probably best if we remove the handlebars `normalize_id` helper registration in `utils.handlesbars.ts` to prevent someone introducing its usage again in a template in the future.